packages/aws-cdk/lib/cli-arguments.ts

```diff
@@ -2,6 +2,7 @@
 // GENERATED FROM packages/aws-cdk/lib/config.ts.
 // Do not edit by hand; all changes will be overwritten at build time from the config file.
 // -------------------------------------------------------------------------------------------
+// istanbul ignore file
```

Comment:
Does this actually do something?

Reply:
No. I forgot to `yarn build` aws-cdk to remove them...